covfee/client/tasks/continuous_annotation/index.tsx

```diff
@@ -99,6 +99,11 @@ const ContinuousAnnotationTask: React.FC<Props> = (props) => {
       needs_upload: false,
     })
 
+  const [showInitialTaskIntructions, setshowInitialTaskIntructions] = useState(
```

Comment:
For consistency, please use camel case -> `setShowInitialTaskIntructions`

Reply:
Fixed in 0772750d37dc1dcc5c873945250f37d1afe68936